Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -101,6 +101,12 @@ public static class MacroHelper
             { "%g2self", Pronoun2self },// Himself/Herself etc...
             { "%g3", Pronoun3 },  // His/Her
             { "%g4", Pronoun4 },  // His/Hers
+            { "%G", PronounCap }, // He/She, first letter capitalized
+            { "%G1", PronounCap }, // He/She, first letter capitalized
+            { "%G2", Pronoun2Cap }, // Him/Her, first letter capitalized
+            { "%G2self", Pronoun2selfCap }, // Himself/Herself, first letter capitalized
+            { "%G3", Pronoun3Cap }, // His/Her, first letter capitalized
+            { "%G4", Pronoun4Cap }, // His/Hers, first letter capitalized
```

Comment:
Please move these below the comment "// DF Unity - new macros:" so they're separated from original game macros